package.json

```diff
@@ -14,6 +14,7 @@
   "devDependencies": {
     "assert": "1.1.1",
     "mocha": "1.18.2",
+    "node-util": "0.0.1",
```

Comment:
https://github.com/defunctzombie/node-url/pull/30/files#diff-1dd241c4cd3fd1dd89c570cee98b79ddR24  you might need to hit load diff